src/LogExpert.UI/Entities/PaintHelper.cs

```diff
@@ -12,7 +11,6 @@
 namespace LogExpert.UI.Entities;
 
 //TOOD: This whole class should be refactored and rethought
-//TODO: This class should not knoow ConfigManager?
```

Comment:
Please return this comment

Reply:
I removed the Configmanager from this class, so it is no longer needed